src/zenml/config/docker_settings.py

```diff
@@ -169,6 +177,8 @@ class DockerSettings(BaseSettings):
     copy_global_config: bool = True
     user: Optional[str] = None
 
+    target_os: Optional[str] = "Linux"
```

Comment:
Any reason we shouldn't use an enum for this?

Reply:
Especially useful as most people would not write "Darwin" instead of "MacOS"

Reply:
Actually I would suggest to remove this from the Docker settings entirely. All our Docker images are linux based, so there's really no reason to expose this.